src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -77,5 +77,11 @@ internal class KnownRoutes
 
         public const string HealthCheck = "/health/check";
         public const string CustomError = "/CustomError";
+
+        public const string SearchParameters = "SearchParameters/";
```

Comment:
I believe this should SearchParameter  not SearchParameter**s**,  not plurarl.

Reply:
Thanks for the catch. Updating.